exercises/concept/troll-the-trolls/.docs/instructions.md

```diff
@@ -31,8 +31,8 @@ Next, define an `Action` enumeration to represent the three permission types: `r
 
 ## 2. Promote trolls only to other trolls.
 
-Every post on the forums saves the `AccountStatus` of the poster in its metadata. 
-Make sure that the trolls can only interact with each other.
+Every post on the forums saves the `AccountStatus` of the poster in its metadata.
+Make sure that posts by trolls are only displayed to other trolls.
```

Comment:
I would add write "the posts", just for consistency.

Reply:
Sorry, I don't understand. Do you mean "... that the posts by trolls ..." instead of "... that posts by trolls ..."?
In the rest of the document "the posts" (with a "the") does not appear, just "posts" alone, e.g. "can read posts", or with a possessive pronoun, e.g. "their posts are".

Reply:
> Do you mean "... that the posts by trolls ..." instead of "... that posts by trolls ..."?

Yeah, that is what I meant, but you can leave it as it is given that "the" doesn't appear before "posts" in the rest of the text, as you noticed.

Reply:
I'll leave that decision to @vaeng

Reply:
I like the less wordy version and would like to keep it without `the`